### PR TITLE
Fixes #110 Built the library from skycoin/libskycoin:develop

### LINKED ISSUE #110: Build C library from skycoin/libskycoin
## Expected Behavior

Since Skycoin 0.25.1 the C client library, SWIG, etc are developed in a separate repository (see skycoin/libskycoin#1) . 

- [x] PySkycoin should use code in `skycoin/libskycoin` repository . 
- [x] Describe releationship between project branches and `skycoin/libskycoin` 's
- [x] Two PR's . One for `develop` (#121) and another one for `v0.25.1`

## Actual Behavior

Submodule tracks `skycoin/skycoin@v0.25.0` .

## Specifications

  - Version: 0.25.0
  - Platform: all
  - Subsystem: all

## Possible Solution
Add `skycoin/libskycoin` as submodule and update GOPATH accordingly in `Makefile`, for instance ...

### PR DESCRIPTION
**Pull request by m4yk3ldev**
Fixes #110 

Changes:
- Submodule change skycoin/skycoin => skycoin/libskycoin@develop
- Changes in url declarations for skycoin/skycoin => skycoin/libskycoin

Does this change need to mentioned in CHANGELOG.md?

Yes

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
-[submodule "skycoin"]
-	path = gopath/src/github.com/skycoin/skycoin
-	url = https://github.com/skycoin/skycoin.git
+[submodule "libskycoin"]
+	path = gopath/src/github.com/skycoin/libskycoin
+	url = https://github.com/skycoin/libskycoin.git
+[submodule "gopath/src/github.com/skycoin/libskycoin"]
```
Comment (olemis) [inline .gitmodules]: add submodule just once

**Review by olemis — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -61,8 +60,7 @@ build-swig: ## Generate Python C module from SWIG interfaces
 	rm -f ./skycoin/skycoin.py
 	rm -f swig/pyskycoin_wrap.c
 	rm -f swig/include/swig.h
-	cp -v gopath/src/github.com/skycoin/skycoin/include/swig.h swig/include/
-	swig -python -w501,505,401,302,509,451 -Iswig/include -I$(INCLUDE_DIR) -outdir ./skycoin/ -o swig/pyskycoin_wrap.c $(LIBSWIG_DIR)/pyskycoin.i
+	swig -python -w501,505,401,302,509,451 -Iswig/include -I$(INCLUDE_DIR) -I$(SKYCOIN_DIR)/include -outdir ./skycoin/ -o swig/pyskycoin_wrap.c $(LIBSWIG_DIR)/pyskycoin.i
```
Comment (olemis) [inline Makefile]: `INCLUDE_DIR = $SKYCOIN_DIR/include`

**Review by olemis — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -82,7 +82,7 @@ before_script:
   # Install PyPA Docker images for building multilinux wheels
 script:
 # Test PySkycoin
-- make test-ci
+- make test
```
Comment (olemis) [inline .travis.yml]: as before

**Review by olemis — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (stdevHan) [reply .gitmodules]: ok, thnx

File: tests/test_cipher_address.py [hunk 4]
```diff
@@ -1,47 +1,5 @@
 import skycoin
 
-def test_TestDecodeBase58Address():
```
Comment (olemis) [inline tests/test_cipher_address.py]: why ? how are them addresses checked for validity ?

Comment (olemis) [inline tests/test_cipher_address.py]: `skycoin/skycoin` is using an exdternal library for `base58` . Please implement this according to this . This function is essential . Needs testing .

**Review by olemis — CHANGES_REQUESTED** [review]

**Comment by olemis** [comment]
Another branch +PR is needed to implement this against `v0.25.1` .

File: tests/test_cipher_address.py (hunk 4 above)
Comment (stdevHan) [reply tests/test_cipher_address.py]: I'll just comment, since 0.25.1 is enabled that function

File: .gitmodules (hunk 1 above)
Comment (stdevHan) [reply .gitmodules]: ok, thnx

**Pull request closed without merging**